res/values-pl/strings.xml

```diff
@@ -151,15 +154,15 @@
     
      <!-- Double-tap-to sleep -->
     <string name="double_tap_sleep_gesture_title">Stuknij dwukrotnie, aby uśpić</string>
-    <string name="double_tap_sleep_gesture_summary">>Podwójnie stuknij pasek statusu, aby uśpić urządzenie</string>
+    <string name="double_tap_sleep_gesture_summary">Podwójnie stuknij pasek statusu, aby uśpić urządzenie</string>
     
     <!-- DoubleTap2Sleep -->
     <string name="double_tap_sleep_lockscreen_title">Dwukrotnie stuknij na ekranie blokady, aby uśpić</string>
-    <string name="double_tap_sleep_lockscreen_sumary">Podwójnie stuknij, aby uśpić urządzenie</string>
+    <string name="double_tap_sleep_lockscreen_summary">Podwójnie stuknij, aby uśpić urządzenie</string>
```

Comment:
you should not change the name of resources, only their values

Reply:
I know, but on English translation exist this string with "summary", **not "sumary"**, also I just adjusted string to English. If you want, i can restore default.
`    <string name="double_tap_sleep_gesture_summary">Double tap on the status bar to put the device to sleep</string>`

Reply:
sorry, this is our mistake (